firedrake/preconditioners/facet_split.py

```diff
@@ -88,10 +88,11 @@ def initialize(self, pc):
             self.set_nullspaces(pc)
             self.work_vecs = self.mixed_opmat.createVecs()
         elif self.subset:
```

Comment:
Configuration member requested below would probably be set before this if-else block and then used for this block, and in the update method below.